src/pages/blog/GHGIntensityPredictCompliance.vue

```diff
@@ -0,0 +1,430 @@
+
+  // New properties
+  results: any = null; // Holds fetched JSON data
+  loading = true; // Controls loading state
+
+  // Lifecycle hook to fetch regression results
+  async mounted(): Promise<void> {
+    const jsonFilePath =
+      '/blog/GHGIntensityPredictCompliance/regression_results_w_covid.json';
+
+    console.log('Fetching JSON from:', jsonFilePath);
+
+    try {
+      const response = await fetch(jsonFilePath);
+
+      if (!response.ok) {
+        throw new Error(`HTTP error! status: ${response.status}`);
+      }
+
+      this.results = await response.json();
+      console.log('Loaded regression results:', this.results);
+    } catch (error) {
+      console.error('Error loading regression results:', error);
+    } finally {
+      this.loading = false;
+    }
+  }
+
+  // Meta information for the page
+  metaInfo(): any {
+    return {
+      title: 'Do High Emitting Buildings Fail to Report?',
+    };
+  }
+}
+</script>
+
+<template>
+  <DefaultLayout>
+    <article class="layout-constrained compliance-analysis blog-post">
+      <g-link to="/blog" class="back-link grey-link"> Back to Blog </g-link>
+
+      <h1 id="main-content" tabindex="-1">
+        Do High Emitters Stop Reporting Emissions Data in the Future?
+      </h1>
+      <p class="sub-title constrained">
+        Analyzing Patterns in Greenhouse Gas Emissions Reporting
+      </p>
+      <p class="publish-time">
+        <!-- TODO: Update publish time when PR is about to get merged -->
+        Published <time datetime="2025-02-02">Feb. 1st, 2025</time>
+      </p>
+
+      <p class="constrained">
+        Many buildings in the publicly available data
+        <a
+          href="https://electrifychicago.net/blog/millions-in-missed-fines"
+          target="_blank"
+          rel="noopener noreferrer"
+          >don't report emissions data.</a
+        >
+        Is there a pattern to which buildings fail to report? Anecdotally, it's
```

Comment:
I'd change this "Anecdotally..." sentence to "Our team has noticed that some high emissions buildings stop reporting, while more efficient buildings tend to keep reporting year after year." There unfortunately has not been real press coverage of this data 😭